lib/firmata.js

```diff
@@ -505,6 +523,12 @@ function Board(port, options, callback) {
     RES_TX3: 0x07,
   };
 
+  this.RESOLUTION = {
+    ADC: null,
```

Comment:
> What happens if the user skips the configurationQuery? Do we need defaults values set here instead?

That's reasonable, we should just use Arduino defaults. (update shortly)


> If we set defaults in firmata.js that param description could change to something like:
> "The data to write to the pin between 0 and this.RESOLUTION.PWM."

Sounds good